flink-table/flink-table-planner/src/test/resources/restore-tests/stream-exec-deduplicate_1/deduplicate-asc/plan/deduplicate-asc.json

```diff
@@ -161,7 +210,7 @@
       "table.exec.mini-batch.size" : "-1"
     },
     "uniqueKeys" : [ 2 ],
-    "isRowtime" : false,
+    "isRowtime" : true,
```

Comment:
Should we also have tests for `isRowtime` `false`? I believe we should cover all properties of the exec node.